code/game/objects/items/grenades/grenade.dm

```diff
@@ -114,3 +111,8 @@
 		owner.visible_message("<span class='danger'>[attack_text] hits [owner]'s [src], setting it off! What a shot!</span>")
 		prime()
 		return TRUE //It hit the grenade, not them
+		
+/obj/item/grenade/afterattack(atom/target as mob|obj|turf|area, mob/user)

```

Comment:
I think there aren't any atoms that aren't atom's children so the whole "as" is superfluous, and besides, this is only useful in the first place for things that prompt you to pick out of a list (read: verbs and popup selectors). Which will never happen here.

Reply:
It's just copy pasta from the beachball. Will remove it soon